attractors/anaconda-project.yml

```diff
@@ -30,6 +30,7 @@ packages: &pkgs
 - hvplot>=0.9.2
 - panel>=1.4.1
 - param>=2.0.1
+- dask>=2024.5.0 # Needed for datashader
```

Comment:
I don't think the comment is clear enough, imagine you come back to this project in a year or two, ask yourself whether you could understand your comment :)

Suggesting `Needed to avoid a warning emitted when datashader internally imports dask.dataframe import`.